components/EditForm/index.js

```diff
@@ -98,11 +99,12 @@ export default function EditForm({ project }) {
   };
 
   if (project === undefined) {
-    return <h2> there is no prject with this id: {id}</h2>;
+    return <h2> there is no project with this id: {id}</h2>;
   }
 
   return (
     <div>
```

Comment:
You could use a fragment here instead and would thus get rid of an unnecessary `div`:
```suggestion
    <>
```

Reply:
thx for the hint...i done it